packages/mc-scripts/src/commands/build.ts

```diff
@@ -0,0 +1,152 @@
+import path from 'path';
+import fs from 'fs-extra';
+import webpack, { type Configuration, type Stats } from 'webpack';
+import chalk from 'chalk';
+import checkRequiredFiles from 'react-dev-utils/checkRequiredFiles';
+import formatWebpackMessages from 'react-dev-utils/formatWebpackMessages';
+import FileSizeReporter from 'react-dev-utils/FileSizeReporter';
+import printBuildError from 'react-dev-utils/printBuildError';
+import { packageLocation as applicationStaticAssetsPath } from '@commercetools-frontend/assets';
+import paths from '../config/paths';
+import createWebpackConfigForProduction from '../config/create-webpack-config-for-production';
+
+const measureFileSizesBeforeBuild =
+  FileSizeReporter.measureFileSizesBeforeBuild;
+const printFileSizesAfterBuild = FileSizeReporter.printFileSizesAfterBuild;
+
+// These sizes are pretty large. We'll warn for bundles exceeding them.
+const WARN_AFTER_BUNDLE_GZIP_SIZE = 512 * 1024;
+const WARN_AFTER_CHUNK_GZIP_SIZE = 1024 * 1024;
+
+async function run() {
```

Comment:
Nothing much changed here, besides wrapping the logic into the `run` function and using more async/await